docs/modules/ROOT/pages/guides/guides-local-multi-node.adoc

```diff
@@ -0,0 +1,106 @@
+= Running a Multi-Node Cluster Locally
+
+== Overview
+Using https://k3d.io[k3d,role=external,window=_blank], a lightweight wrapper to run Rancher’s minimal Kubernetes
+distribution https://github.com/k3s-io/k3s[k3s,role=external,window=_blank], we can run a multi-node cluster within
+Docker for local development. This allows us to verify that we're able to execute particular components on particular
+nodes or node types without necessarily needing access to a cluster with multiple physical nodes. One use case is the
+execution of machine learning inference and training components on nodes with GPU-acceleration available. This
+technique also allows us to verify that scaling is occurring appropriately. For example, we can verify that PySpark
+data delivery pipeline workers are scaling horizontally to improve throughput.
+
+=== Setup
+1. In Rancher Desktop settings, set Container Runtime to `dockerd(moby)`
+2. Install k3d choosing from the following options:
+* `wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash`
+* `curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash`
+* *macOS:* `brew install k3d`
```

Comment:
Q: Should this be macOS, or should it be Homebrew?

Reply:
I think mac is right here